README.md

```diff
@@ -0,0 +1,15 @@
+                                       Cohort 1 Training project for backend
+
+Aint Board Game - Backend
+
+Node.js
+Npm
+Prisma
+Express
+
+
+Getting Started First, run the development server:
+
+npm run start 
```

Comment:
youre missing npm install

also how do we setup the DB?

Reply:
sorry I missed this ..adding the instructions